kubernetes-deployment/maintainlib/repair.py

```diff
@@ -0,0 +1,87 @@
+# Copyright (c) Microsoft Corporation
+# All rights reserved.
+#
+# MIT License
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+# documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+# to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+# The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+# BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+# NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+# DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+import yaml
+import os
+import sys
+import subprocess
+import jinja2
+import argparse
+import paramiko
+import common
+
+
+class repair:
+
+    """
+    An class to reinstall
+    """
+
+    def __init__(self, cluster_config, node_config, clean):
+
+        self.cluster_config = cluster_config
+        self.node_config = node_config
+        self.maintain_config = common.load_yaml_file("maintain.yaml")
+        self.jobname = "repair"
+        self.clean_flag = clean
+
+
+
+    def prepare_package(self):
+
+        common.maintain_package_wrapper(self.cluster_config, self.maintain_config, self.node_config, self.jobname)
+
+
+
+    def delete_packege(self):
```

Comment:
>delete_packege [](start = 8, length = 14)

please add unit test that cover prepare and delete package

Reply:
Done in common